core/src/apps/management/apply_settings.py

```diff
@@ -10,11 +10,32 @@
 if TYPE_CHECKING:
     from trezor.enums import SafetyCheckLevel
     from trezor.messages import ApplySettings, Success
+    from trezor.ui import ProgressLayout
 
 
 BRT_PROTECT_CALL = ButtonRequestType.ProtectCall  # CACHE
 
 
+async def _load_homescreen(length: int) -> bytes:
```

Comment:
nit: it returns a `bytearray`, right?

Reply:
right, thanks, fd1419a708062560e063c3d26ac3e20da93520ac